code/modules/mafia/roles/roles.dm

```diff
@@ -46,11 +51,43 @@
 		role_unique_actions -= abilities
 
 /datum/mafia_role/Destroy(force, ...)
+	send_back_to_body()
```

Comment:
I'm just not too entirely sure at what point we should have it in. I was thinking of putting it in mafia's ``end_game()``, but now since it's a component I'm not sure it'll work out that well.